assets/src/edit-story/components/canvas/useCanvasKeys.js

```diff
@@ -35,15 +35,39 @@ const MOVE_COARSE_STEP = 10;
  */
 function useCanvasKeys(ref) {
   const {
-    actions: {
-      addElements,
-      arrangeSelection,
-      clearSelection,
-      deleteSelectedElements,
-      updateSelectedElements,
-    },
-    state: { selectedElements },
-  } = useStory();
+    selectedElements,
+    addElements,
+    arrangeSelection,
+    clearSelection,
+    deleteSelectedElements,
+    updateSelectedElements,
+  } = useStory(
+    ({
+      // eslint-disable-next-line no-shadow
+      state: { selectedElements },
+      actions: {
+        // eslint-disable-next-line no-shadow
```

Comment:
FYI, you can just do `eslint-disable no-shadow` somewhere and `eslint-enable no-shadow` afterwards instead of repeating this 7 times.

Reply:
Updated, for selectors that have more than 2 lines of disabled lint rules.